### PR TITLE
Use https for all links

### PR DESCRIPTION
**Pull request by dns2utf8**
Hi all

This is a followup to https://github.com/gtk-rs/gtk/pull/1054

Cheers,
Stefan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 Gdk bindings for Rust.
 
-- [Gtk-rs project site](http://gtk-rs.org/)
+- [Gtk-rs project site](https://gtk-rs.org/)
 
-- [Online documentation](http://gtk-rs.org/docs/)
+- [Online documentation](https://gtk-rs.org/docs/)
```
Comment (GuillaumeGomez) [inline README.md:7]: https://gtk-rs.org/docs-src/

Comment (dns2utf8) [reply README.md]: Thanks!

File: Cargo.toml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "gdk"
 license = "MIT"
-homepage = "http://gtk-rs.org/"
+homepage = "https://gtk-rs.org/"
 authors = ["The Gtk-rs Project Developers"]
 keywords = ["gdk", "gtk", "gtk-rs", "gnome", "GUI"]
 readme = "README.md"
-documentation = "http://gtk-rs.org/docs/gdk/"
+documentation = "https://gtk-rs.org/docs-src/gdk/"
```
Comment (GuillaumeGomez) [inline Cargo.toml:8]: Bad change. ;)

Comment (dns2utf8) [reply Cargo.toml]: Thank you!

**Comment by dns2utf8** [comment]
Fixed it

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**